doc/compiler/jitserver/Problem.md

```diff
@@ -86,3 +88,6 @@ we usually go for the latter solution, as it is easier to implement.
 - **Missing VM information**: client needs to inform the server about some VM options that can affect compilation, e.g. GC mode.
 If some information about the VM setup is missing, server might compile something that will not run correctly. Usually, such information
 should go into the `VMInfo` object on the server.
+
+
+For debugging AOT Cache related issues, there are options available at JITServer client to exclude methods from getting stored/loaded from the JITServer AOT cache. With the `-Xaot:jitserverLoadExclude={<method_name_regex>}` and `-Xaot:jitserverStoreExclude={<method_name_regex>}` options, clients can selectively exclude methods from AOT cache load or store in a connected JITServer. This can be useful to determine whether a problem with a given method is caused by an AOT compiler issue or an AOT cache method serialization/deserialization issue. Note that these options should be specified on the client side.
```

Comment:
For debugging JITServer AOT Cache related issues, there are two options that can be useful to determine whether a problem with a given method is caused by an AOT compiler issue or by a JITServer AOT cache method serialization/deserialization issue:
* -Xaot:jitserverLoadExclude={<method_name_regex>}` option prevents the specified  method(s) from being loaded from the JITServer AOT cache.
* `-Xaot:jitserverStoreExclude={<method_name_regex>}` option prevents the specified  method(s) from being stored to the JITServer AOT cache.

Note that these options should be specified on the client side.